src/components/swapv2/TokenInfoV2/SingleTokenInfo.tsx

```diff
@@ -0,0 +1,200 @@
+}
+
+const SwapInstruction = styled.div`
+  margin-top: 16px;
+  color: ${({ theme }) => theme.subText};
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 24px;
+`
+
+export function HowToSwap({
+  fromCurrency,
+  toCurrency,
+  fromCurrencyInfo,
+  toCurrencyInfo,
+  expandedOnMount,
+}: {
+  fromCurrency: Currency | undefined
+  toCurrency: Currency | undefined
+  fromCurrencyInfo: TokenInfo
+  toCurrencyInfo: TokenInfo
+  expandedOnMount?: boolean
+}) {
+  const theme = useTheme()
+
+  if (!fromCurrency || !toCurrency || !fromCurrencyInfo || !toCurrencyInfo) return null
+  const symbol1 = fromCurrency.symbol
+  const symbol2 = toCurrency.symbol
+  const name1 = fromCurrency.name
+  const name2 = toCurrency.name
+
+  const fromName = formatString(fromCurrencyInfo.name || name1)
+  const toName = formatString(toCurrencyInfo.name || name2)
+
+  return (
+    <BlockWrapper
+      expandedOnMount={expandedOnMount}
+      header={
+        <Text
+          as="h2"
+          sx={{
+            color: theme.subText,
+            fontSize: '20px',
+            fontWeight: 500,
+          }}
+        >
+          How to swap {symbol1} to {symbol2}?
+        </Text>
+      }
+    >
+      <SwapInstruction>
+        <Text as="span">
+          {fromName} ({symbol1}) can be exchanged to {toName} ({symbol1} to {symbol2}) on KyberSwap, a cryptocurrency
+          decentralized exchange. By using KyberSwap, users can trade {symbol1} to {symbol2} on networks at the best
+          rates, and earn more with your {symbol1} token without needing to check rates across multiple platforms.
+        </Text>
+      </SwapInstruction>
+    </BlockWrapper>
+  )
+}
+
+const SingleTokenInfo = ({
+  data: tokenInfo,
+  currency,
+  loading,
+  expandedOnMount,
+}: {
+  data: TokenInfo
+  currency?: Currency
+  loading: boolean
+  expandedOnMount?: boolean
+}) => {
+  const description = replaceHtml(tokenInfo?.description?.en)
+
+  const ref = useRef<HTMLParagraphElement>(null)
+
+  const symbol = currency?.symbol
+  const currencyName = tokenInfo.name || currency?.name
+
+  const listField = [
+    { label: 'Price', value: tokenInfo.price ? formattedNum(tokenInfo.price.toString(), true) : NOT_AVAILABLE },
+    {
+      label: 'Market Cap Rank',
+      value: tokenInfo.marketCapRank ? `#${formattedNum(tokenInfo.marketCapRank.toString())}` : NOT_AVAILABLE,
+    },
+    {
+      label: '24H Volume',
+      value: !tokenInfo.tradingVolume
+        ? NOT_AVAILABLE
+        : isMobile
+        ? formatDollarAmount(tokenInfo.tradingVolume, 2).toUpperCase()
+        : formattedNum(tokenInfo.tradingVolume.toString(), true),
+    },
+  ]
+  return (
+    <BlockWrapper
+      expandedOnMount={expandedOnMount}
+      header={
+        <Flex alignItems="center">
+          <CurrencyLogo currency={currency} size="24px" style={{ marginRight: 10 }} />
+          <AboutText>
+            {/* About Usdt (Tether(...)) => Usdt (Tether) */}
```

Comment:
Can you check this comment, the result's not gonna be `Usdt (Tether)`, it's `Usdt` instead

Reply:
![image](https://user-images.githubusercontent.com/19758667/180702265-61dfaa78-19ce-4cdc-b63e-49092e579095.png)
It's correct. This file was moved to a new folder, so all changes were marked new 😬